deepfence_frontend/apps/dashboard/src/components/GoBack.tsx

```diff
@@ -0,0 +1,22 @@
+import { IconContext } from 'react-icons';
+import { HiArrowSmLeft } from 'react-icons/hi';
+import { useNavigate } from 'react-router-dom';
+import { Button } from 'ui-components';
+
+export const GoBack = () => {
```

Comment:
we should no use navigate(-1), it breaks when user opens a page via a direct link(for example pdf report). always use full paths, It's a bit verbose but doesn't break.